test/e2e/ginkgo/multiple_cluster_deployment_test.go

```diff
@@ -469,30 +475,28 @@ var _ = ginkgo.Describe("end to end testing", func() {
 		})
 
 		ginkgo.It("available replicas count", func(ctx context.Context) {
-			util.CreateStatusCollector(ctx, ksWds, countAvailableReplicasStatusCollectorName,
+			availableReplicasCEL := ksapi.Expression("returned.status.availableReplicas")
+			util.CreateStatusCollector(ctx, ksWds, sumAvailableReplicasStatusCollectorName,
 				ksapi.StatusCollectorSpec{
-					GroupBy: []ksapi.NamedExpression{{
-						Name: "num-available",
-						Def:  "obj.availableReplicas",
-					}},
 					CombinedFields: []ksapi.NamedAggregator{{
-						Name: "count",
-						Type: ksapi.AggregatorTypeCount,
+						Name:    "sum",
+						Subject: &availableReplicasCEL,
+						Type:    ksapi.AggregatorTypeSum,
 					}},
 					Limit: 10,
 				})
 
-			testAndStatusCollection[0].StatusCollectors = []string{countAvailableReplicasStatusCollectorName}
+			testAndStatusCollection[0].StatusCollectors = []string{sumAvailableReplicasStatusCollectorName}
 			util.CreateBindingPolicy(ctx, ksWds, bpName, clusterSelector, testAndStatusCollection)
 
 			cs := util.GetCombinedStatus(ctx, ksWds, wds, ns, workloadName, bpName)
 
 			// Validate CombinedStatus results
 			gomega.Expect(len(cs.Results)).To(gomega.Equal(1))
-			gomega.Expect(cs.Results[0].Name).To(gomega.Equal(countAvailableReplicasStatusCollectorName))
+			gomega.Expect(cs.Results[0].Name).To(gomega.Equal(sumAvailableReplicasStatusCollectorName))
 
 			gomega.ExpectWithOffset(1, len(cs.Results[0].ColumnNames)).To(gomega.Equal(1))
```

Comment:
Not introduced in this PR, but: why is there an offset of 1 here? I see inconsistent offsets within `ginkgo.Context("combined status testing", func() {...})`. If I understand correctly, an offset of 1 should be used in a function _called from_ a subject node like those in this ginkgo.Context.